ecl/hqlcpp/hqlstmt.cpp

```diff
@@ -824,6 +824,38 @@ HqlExprAssociation * BuildCtx::queryFirstCommonAssociation(AssocKind searchKind)
 }
 
 
+void BuildCtx::walkAssociations(AssocKind searchMask, IAssociationVisitor & visitor)
```

Comment:
@ghalliday I don't see walkAssociations used anywhere.  Is this function required for tracking activity time?